### PR TITLE
Update snapshot versions and fix build failures

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Comment by mareknovotny** [comment]
it seems ok, do we have setup already PR builder CI job to show results for it here?
@mbiarnes, @jiripetrlik  ?

**Comment by jiripetrlik** [comment]
@mareknovotny Seems, that there is no PR check yet.
@mbiarnes Can you please add PR check here?

File: kie-remote/src/main/java/org/kie/remote/CommonConfig.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
     private static Properties config;
 
-    public static Properties getStatic() {
+    public static synchronized Properties getStatic() {
```
Comment (danielezonca) [inline kie-remote/src/main/java/org/kie/remote/CommonConfig.java]: Why this change? Is this method invoked by multiple thread concurrently?

File: kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java [hunk 2]
```diff
@@ -36,11 +36,10 @@
 import org.kie.remote.message.ListKieSessionObjectMessage;
 import org.kie.remote.message.VisitableMessage;
 import org.kie.remote.message.VisitorMessage;
+import org.kie.remote.util.SerializationUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import static org.kie.remote.util.SerializationUtil.deserialize;
```
Comment (danielezonca) [inline kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java]: Is this related to -Dfull changes?

File: kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java [hunk 3]
```diff
@@ -23,13 +23,11 @@
 import org.kie.remote.RemoteCepKieSession;
 import org.kie.remote.TopicsConfig;
 
-import static org.kie.remote.impl.producer.RemoteKieSessionImpl.DEFAULT_ENTRY_POINT;
```
Comment (danielezonca) [inline kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java]: Same question as above

File: kie-remote/src/main/java/org/kie/remote/CommonConfig.java (hunk 1 above)
Comment (jiripetrlik) [reply kie-remote/src/main/java/org/kie/remote/CommonConfig.java]: @danielezonca 
For some reason one our bug hunting plugin fails on this because it is not thread safe for some reasons. Try it with -Dfull parameter. Build fails if method is not "synchronized".

File: kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java (hunk 2 above)
Comment (jiripetrlik) [reply kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java]: No, it is not probably some automatic IDE change during refactoring. Do you see it as a problem?

File: kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java (hunk 3 above)
Comment (jiripetrlik) [reply kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java]: No, it is not probably some automatic IDE change during refactoring. Do you see it as a problem?

**Comment by mbiarnes** [comment]
Jenkins retest this please

File: kie-remote/src/main/java/org/kie/remote/CommonConfig.java (hunk 1 above)
Comment (danielezonca) [reply kie-remote/src/main/java/org/kie/remote/CommonConfig.java]: ok

File: kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java (hunk 2 above)
Comment (danielezonca) [reply kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java]: It is not a problem but I would like to have an agreement about `static import` and similar otherwise it will become a "refactor" war in every PR :smile: 

I.e. we have many other import static in this repo:
`./kie-remote/src/main/java/org.kie.remote/impl/producer/EventProducer.java:import static org.kie.remote.util.SerializationUtil.serialize;
./hacep/src/main/java/org/kie/hacep/consumer/DroolsConsumerHandler.java:import static org.kie.remote.util.SerializationUtil.deserialize;
[...]`

File: kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java (hunk 3 above)
Comment (danielezonca) [reply kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java]: Same answer :smile:

File: kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java (hunk 2 above)
Comment (danielezonca) [reply kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java]: Btw it is a minor comment, considering that this PR is important to have master stable fine for me to merge and then we can discuss about that offline

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
@mareknovotny I don't understand why this is not starting?

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by jiripetrlik** [comment]
PR was replaced because of PR check.
https://github.com/kiegroup/openshift-drools-hacep/pull/6
